bikeshare_app/views.py

```diff
@@ -107,6 +107,13 @@ def register():
         # Allow user through if check passed, else log attempt and return home
         return render_template('register.html')
 
+@app.route('/checkin', methods=['GET'])
```

Comment:
I believe I did the same for the sole purpose of an implication that `POST` will one day be incorporated as well. A `# TODO` will probably suffice instead?